### DISCUSSION TITLE
GROD Misorientation Axis

### DISCUSSION THREAD

**Discussion by NovakEvana** [mtex-toolbox/mtex#1721] [Ask Anything] [answered]
Hi everyone,

I was trying to generate the GROD misorientation axis by using the crystal coordinates following [this instruction](https://mtex-toolbox.github.io/EBSDGROD.html). However, the result returned a pole figure instead of an IPF triangle type as demonstrated in the instruction. Any idea on this would be very much appreciated. Thank you.

```
startup_mtex;
CS = {... 
  'notIndexed',...
  crystalSymmetry('m-3m', [3.6599 3.6599 3.6599], 'mineral', 'Austenite FCC', 'color', [1 0 0]),...
  crystalSymmetry('m-3m', [2.8660 2.8660 2.8660], 'mineral', 'Martensite BCC', 'color', [0 1 0])};

setMTEXpref('xAxisDirection','south');
setMTEXpref('zAxisDirection','outOfPlane');

ebsd = EBSD.load(fname,CS,'interface','ang',...
  'convertSpatial2EulerReferenceFrame');

 [grains,ebsd.grainId] = calcGrains(ebsd('indexed'),'angle',5*degree)
ebsd(grains(grains.grainSize < 2)) = [];
[grains,ebsd.grainId] = calcGrains(ebsd('indexed'),'angle',5*degree)
% ... and smooth the grain boundaries
grains = smooth(grains,5);
% denoise the orientations
F = halfQuadraticFilter;
ebsd = smooth(ebsd,F,grains,'Fill');

% take the avarage of the misorientation angles for each grain
grod = ebsd.calcGROD(grains);
axCrystal = grod.axis;
plot(axCrystal,'contourf','fundamentalRegion','antipodal','figSize','small')
mtexColorbar('title','distribution of misorientation axes in mrd')
```

![image](https://user-images.githubusercontent.com/117882416/213534746-bf9b42f9-3eff-45cd-86a9-d765a4d899cc.png)

**Comment by kilir** [comment, marked as answer]
Hi,
of what type is grod in your case?
And what happens if you use a single phase only?

If your output is a rotation, the axis is a vector3d  and will occupy the entire hemisphere, if it is a misorientation, axes will  be of type Miller and obey crystal symmetry. 

Cheers,
Rüdiger

**Reply by NovakEvana** [reply]
Sorry for the late response. I changed to a single phase and it worked. Thanks Rudiger!
